apps/backend/src/api/audit.py

```
@@ -0,0 +1,197 @@
+    """List data collection audit entries with filtering and pagination."""
+    try:
+        # Parse comma-separated filter parameters
+        audit_filter = DataCollectionFilter(
+            device_ids=[UUID(id.strip()) for id in device_ids.split(",")] if device_ids else None,
+            data_types=data_types.split(",") if data_types else None,
+            collection_methods=collection_methods.split(",") if collection_methods else None,
+            statuses=statuses.split(",") if statuses else None,
+            errors_only=errors_only,
+            cache_hit_only=cache_hit_only,
+        )
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)